.github/workflows/validate-header-guards.yml

```diff
@@ -34,3 +34,6 @@ jobs:
       - name: checknames
         run: |
           python tools/scripts/checkDuplicateFilenames.py
+      - name: checklicense
```

Comment:
As a side note, this doesn't have to be so simple. The value for `name` key is a string, so you can write more descriptive name, e.g. "Check that files use proper license header", like so

```yaml
name: Check that files use proper license header
```